website/docs/apps/autoplot_app/README.md

```diff
@@ -110,14 +110,15 @@ The control panel is accessed by clicking on the gear icon located in the upper
 right corner.
 ![gearing](https://github.com/farm-ng/amiga-dev-kit/assets/133177230/f09d117f-1026-4036-b2f0-20a557f83c9a)
 
-Our current Firmware release
-[**amiga-dash-v0.5.0**](https://github.com/farm-ng/amiga-dev-kit/releases/tag/amiga-dash-v0.5.0) supports
-up to four H-Bridge devices that can be latched or momentarily controlled.
-They can also be coupled for synchronous operation.
+Starting with dashboard firmware release
+[**amiga-dash-v0.5.0**](https://github.com/farm-ng/amiga-dev-kit/releases/tag/amiga-dash-v0.5.0),
+there is support for up to four H-Bridge devices.
+These devices can be either latched or momentarily controlled,
+and can optionally be coupled for synchronous operation.
 
 ![Screenshot from 2024-05-02 18-05-04](https://github.com/farm-ng/amiga-dev-kit/assets/133177230/ffb45918-ed86-497d-a72b-5892d75053ab)
 
-Up to four PTO Devices are supported and can be remotely selected for operation.
+Up to two PTO Devices are also supported and can be remotely selected for operation.
```

Comment:
I thought it was four. Did we intentionally decreased the number of supported PTOs?

Reply:
Only two are enabled in the firmware, to reduce the memory footprint